server/src/test/java/org/elasticsearch/gateway/PersistedClusterStateServiceTests.java

```diff
@@ -1392,7 +1391,7 @@ public void testLimitsFileCount() throws IOException {
         }
     }
 
-    public void testOverrideLuceneVersion() throws IOException {
+    public void testOverrideNodeVersion() throws IOException {
```

Comment:
@grcevski I see you added this test, do you agree with the rename? I think naming is what made the update for v9 below end up on Search Foundations backlog, but I think 1. it's not a search foundations task 2. nothing needs to be done. Can you double check please?